memory/ndctl.py

```diff
@@ -186,6 +186,30 @@ def test_dimms(self):
             self.fail('Failed to fetch DIMMs')
         self.log.info('Available DIMMs: %s', vals)
 
+    @avocado.fail_on(pmem.PMemException)
+    def test_dimm_health(self):
+        """
+        Test the dimm health
+        """
+        dimms = self.plib.run_ndctl_list('-DH')
+        if not dimms:
+            self.fail('Failed to fetch DIMMs')
+        for dimm in dimms:
+            health = self.plib.run_ndctl_list_val(dimm, 'health')
+            if not health:
+                self.cancel("kernel/ndctl does not support health reporting")
+            if 'life_used_percentage' in health:
+                if health['life_used_percentage'] != 0:
+                    self.fail("DIMM has not been used, but reporting says otherwise")
```

Comment:
some how message not looks proper 
either it can be 
`"DIMM has not been used  but reporting says otherwise"`
or 
`"DIMM has not been used , reporting says otherwise"`

Reply:
ok